<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartCalcView</class>
 <widget class="QMainWindow" name="SmartCalcView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>309</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>523</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>523</width>
    <height>309</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartCalcView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>523</width>
      <height>302</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>523</width>
      <height>302</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>523</width>
      <height>302</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
color: rgb(0,0,0);
background-color: #828282;
border-color: #BEBEBE;
 border-style: solid;
 border-width:1px;
 border-radius:10px;
}
QPushButton:hover {
color: rgb(230, 230, 230);
background-color: #606060;
border-color: #BEBEBE;
border-style: solid;
border-width:1px;
border-radius:10px;
}
QPushButton:pressed {
    color: rgb(144, 143, 146);
background-color: rgb(16, 94, 46);
border-color: rgb(144, 143, 146);
 border-style: solid;
 border-width:2px;
 border-radius:10px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QRadioButton" name="radioButton_exp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(102, 255, 0)</string>
           </property>
           <property name="text">
            <string>Выражение</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_exp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(102, 255, 0);</string>
           </property>
           <property name="maxLength">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QRadioButton" name="radioButton_x">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(102, 255, 0)</string>
           </property>
           <property name="text">
            <string>x = </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_x">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(102, 255, 0);</string>
           </property>
           <property name="maxLength">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>AC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear_one_sym">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>⌫</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_sin">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>sin</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_tan">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>tan</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_asin">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>asin</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_sqrt">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>sqrt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_log">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>log</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_cos">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>cos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_acos">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>acos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_atan">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>atan</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_ln">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>ln</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_pow">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>^</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_openb">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>(</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_1">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_0">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton_closeb">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_x">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_mod">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_9">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_point">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QPushButton" name="pushButton_div">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>/</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_mult">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>*</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_minus">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_plus">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_equals">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>=</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Кредитный калькулятор</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
